pkg/helpers/helpers.go

```diff
@@ -119,17 +119,36 @@ func UpdateSubmarinerConfigStatus(
 	return updatedStatus, updated, err
 }
 
+func IsSubmarinerEnvPrepared(
```

Comment:
This is now only called by the spoke controller.  I'd prefer to move it local rather than lumping it in a general helpers "grab bag" (I'm gradually moving functions out of here).

That said, is it really necessary in the spoke controller?

Reply:
changed to local. Ya check is required to prevent cloud-prepare being called for each reconcile.